# **Question:**


##### Which class had the most survivors?




## **Study Description:**


##### This will require a census study because it needs to gather data from the whole passenger population of the Titanic, and I have access to that entire population in my dataset.


###### Here is the data that needs to be collected from the titanic.csv file:
###### - The whole passenger population's class
###### - Whether they survived or not

In [4]:
import pandas as pd

In [5]:
titanic = pd.read_csv("titanic.csv")

In [32]:
class_1 = titanic[titanic["Pclass"] == 1]
class1_survivors = class_1["Survived"].sum()
print(class1_survivors)

136


In [30]:
class_2 = titanic[titanic["Pclass"] == 2]
class2_survivors = class_2["Survived"].sum()
print(class2_survivors)

87


In [33]:
class_3 = titanic[titanic["Pclass"] == 3]
class3_survivors = class_3["Survived"].sum()
print(class3_survivors)

119


In [26]:
print(max(class1_survivors, class2_survivors, class3_survivors))

136


# Conclusion
##### Since class 1 had 136 survivors, which was the most out of all three classes, class 1 had the most survivors on the *Titanic*